### ISSUE TITLE
Supermarket missing latest version

### ISSUE THREAD

**Issue by ctbarrett** [xhost-cookbooks/system#60]
Could you please bump the version on Supermarket to `0.12.0`? It's still showing `0.11.3` as the latest.

**Comment by flaccid** [comment]
Supermarket is rejecting my key and re-issue doesn't help. Where can I get support for that?

**Comment by fletchowns** [comment]
@flaccid were you ever able to fix this one? Maybe somebody in the chef slack channel can help: http://community-slack.chef.io/

**Comment by flaccid** [comment]
Nope. I am on there and asked quite a few times with no luck in the past.

**Comment by fletchowns** [comment]
@flaccid Dang :(

Maybe try a post on https://discourse.chef.io/ ?

**Comment by hifzan-a3k** [comment]
if you have extra spaces at the bottom of metadata or Berksfile, remove them it should work. at least this is how it was fixed for me 🤷
